book/2.json

```diff
@@ -1,42 +1,42 @@
-    {"text":"\nDiamond chestplate ","color":"dark_gray"},
+    {"text":"\nDiamond chestplate ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, diamond chestplates\nwill be converted back into diamonds"}},
     {"storage":"uhc_pack:text","nbt":"Icon.DiamondChestplate","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 115"}},
-    {"text":"\nStrong potions ","color":"dark_gray"},
+    {"text":"\nStrong potions ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, tier 2 potions will\nbe converted into tier 1 potions"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Potion","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 109"}},
-    {"text":"\nSuspicious stew ","color":"dark_gray"},
+    {"text":"\nSuspicious stew ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, suspicious stews are\nconverted into mushroom stew"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Stew","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 110"}},
-    {"text":"\nPassive mob AI ","color":"dark_gray"},
+    {"text":"\nPassive mob AI ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, passive mobs\nwill not have any AI\n\nThis excludes tamable mobs,\nsuch as wolves"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Passive","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 111"}},
-    {"text":"\nPhantom spawning ","color":"dark_gray"},
+    {"text":"\nPhantom spawning ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, phantoms will\not be able to spawn naturally"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Phantom","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 112"}},
-    {"text":"\nTeam joining ","color":"dark_gray"},
+    {"text":"\nTeam joining ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"When disabled, players are not\nable to switch teams by themselves"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Joining","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 113"}},
-    {"text":"\nNight vision ","color":"dark_gray"},
+    {"text":"\nNight vision ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Give night vision to spectators"}},
```

Comment:
It actually gives it to all players. Spectators will have night vision regardless of this setting